draft/2022-04-13-this-week-in-rust.md

```diff
@@ -230,6 +217,7 @@ Rusty Events between 2022-04-06 - 2022-05-04 🦀
     * [**Rust Bristol 🦀 Kickoff (2/2)**](https://www.meetup.com/rust-bristol/events/284704573/)
 
 ### North America
+
 * 2022-04-13 | Atlanta, GA, US | [Rust ATL](https://www.meetup.com/Rust-ATL/)
     * [**Grab a beer with fellow Rustaceans**](https://www.meetup.com/Rust-ATL/events/pczdssydcgbrb/)
 * 2022-04-14 | Columbus, OH, US | [Columbus Rust Society](https://www.meetup.com/columbus-rs/)
```

Comment:
Might we wish to add: 5/11 (Atlanta GA) [Grab a beer with fellow Rustaceans](https://www.meetup.com/Rust-ATL/events/pczdssydchbpb/)